js/components/sortable-densities.js

```diff
@@ -10,7 +10,7 @@ function hsvToBackground (hsvColor) {
 }
 
 const SortableItem = SortableElement(({plateInfo}) =>
-  <li className='density-button-container' style={hsvToBackground(plateInfo.color)} key={plateInfo.id}>
+  <li className='density-button-container' style={hsvToBackground(plateInfo.color)}>
```

Comment:
@pjanik, I *believe* this key is unnecessary. I don't see any React warnings in my console, but I remember that you might have, so let me know how this looks for you.